docs/api-testing/composer.md

```diff
@@ -427,8 +419,8 @@ To create an object of data:
 
 Any of the following methods can be used in a request body.
 
-* **Content-Type:** The content-type of the body (application/json in this example).
-* **Content:** The body of the request.
+* **Content Type:** The content-type of the body (application/json in this example).
```

Comment:
@aliciakrebs the label is still "Content-Type"

Reply:
fixed